recipes.json

```diff
@@ -1,43 +1,109 @@
 {
     "carbonara": {
         "ingredients": {
-            "egg": 2,
-            "parmesan cheese": 1,
-            "spaghetti": 1,
-            "bacon": 4
+            "egg": [
+                2,
+                "unit"
+            ],
+            "parmesan cheese": [
+                200,
+                "Gramms"
+            ],
+            "spaghetti": [
+                1,
+                "Gramms"
+            ],
+            "bacon": [
+                4,
+                "Rashers"
+            ]
         },
         "portions": 2
     },
     "bacon and egg muffins": {
         "ingredients": {
-            "egg": 1,
-            "english muffins": 1,
-            "bacon": 2,
-            "butter": 1
+            "egg": [
+                1,
+                "Unit"
+            ],
+            "english muffins": [
+                1,
+                "Unit"
+            ],
+            "bacon": [
+                2,
+                "Rashers"
+            ],
+            "butter": [
+                1,
+                "tsp"
+            ]
         },
         "portions": 1
     },
     "beef keema": {
         "ingredients": {
-            "rice": 1,
-            "beef mince": 1,
-            "curry powder": 1,
-            "carrot": 2,
-            "ginger": 1,
-            "garlic clove": 3,
-            "coriander": 1,
-            "chicken stock": 1
+            "rice": [
+                150,
+                "Gramms"
+            ],
+            "beef mince": [
+                300,
+                "Gramms"
+            ],
+            "curry powder": [
+                1,
+                "Tbsp"
+            ],
+            "carrot": [
+                2,
+                "Unit"
+            ],
+            "ginger": [
+                15,
+                "Gramms"
+            ],
+            "garlic clove": [
+                3,
+                "Gramms"
+            ],
+            "coriander": [
+                10,
+                "Gramms"
+            ],
+            "chicken stock": [
+                1,
+                "Unit"
```

Comment:
is this 1 chicken stock cube? or would it be better as like "250 ml"?

Reply:
as above, thinking stock cube/stock pot, rather than amount of liquid.